lib/airrecord/table.rb

```diff
@@ -25,21 +25,30 @@ class << self
       def deprecate_symbols
         warn Kernel.caller.first + ": warning: Using symbols with airrecord is deprecated."
       end
-      
+
       def client
         @@clients ||= {}
         @@clients[api_key] ||= Client.new(api_key)
       end
 
-      def has_many(name, options)
-        @associations ||= []
-        @associations << {
-          field: name.to_sym, # todo: deprecate_symbols
-        }.merge(options)
+      def has_many(method_name, options)
+        # define an assocation getter
+        define_method(method_name.to_sym) do
+          # Get association ids in reverse order, because Airtable’s UI and API
+          # sort associations in opposite directions. We want to match the UI.
+          ids = (self[options.fetch(:column)] || []).reverse
+          table = Kernel.const_get(options.fetch(:class))
+          options[:single] ? table.find(ids.first) : table.find_many(ids)
+        end
+
+        # define an assocation setter
```

Comment:
Same for this one, but the one with a note on why do it in reverse is excellent.

Reply:
Do you think association setters should, like getters, reverse the array order? I think they probably should, so that working with associations always behaves consistently with Airtable's UI, while working with raw columns behaves consistently with Airtable's API.